it/test/controllers/ClaimToAdjustPoaControllerISpec.scala

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright 2017 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers
+
+import models.claimToAdjustPoa.{ClaimToAdjustPoaRequest, MainIncomeLower}
+import play.api.http.Status._
+import play.api.libs.json.Json
+import test.helpers.{ComponentSpecBase, WiremockHelper}
+
+class ClaimToAdjustPoaControllerISpec extends ComponentSpecBase {
```

Comment:
+ add cases for authorised / no authorised as well

Reply:
Done